admin/tests/acceptance/authenticated/users/get_test.js

```diff
@@ -249,4 +249,38 @@ module('Acceptance | authenticated/users/get', function (hooks) {
       assert.dom(screen.getByText('Modifier le rôle')).exists();
     });
   });
+
+  module('when administrator clicks on certification centers tab', function () {
+    test('should display user’s certification centers', async function (assert) {
+      // given
+      const certificationCenter = this.server.create('certification-center', {
+        name: 'Centre Kaede',
+        externalId: 'ABCDEF12345',
+        type: 'SCO',
+      });
+      const certificationCenterMembership = this.server.create('certification-center-membership', {
+        certificationCenter,
+      });
+      const user = server.create('user', {
```

Comment:
**question**
Pourquoi utilise-t-on parfois `this.server.create` et parfois `server.create` ?